templates/graph/index.html

```diff
@@ -129,7 +130,7 @@ <h4 class="modal-title" id="myModalLabel">Save Layout</h4>
                             <br>
                             <div class="form-group">
                                 <button name="saveLayoutBtn" id="saveLayoutBtn"
-                                        class="btn btn-success sidebar-nav-pills" data-target="#layoutEditorSideBar">Save
+                                        class="btn btn-success sidebar-nav-pills" data-target="#defaultSideBar">Save
```

Comment:
This might have an undesirable impact on `#layoutEditorSidebar` modal. 
Are we completely getting rid of the existing way layouts are saved? 
@lohani2280  What happens when a user enters **Use Layout Editor** and then exits from the editor ?? Will they be greeted with the `Save and exit/ Exit without saving/Close` options ?

Reply:
I did not change anything with the `Save on Exit Layout Modal` in the `graph/index.html` file. Thus, when a user enters **Use Layout Editor** and then exits from the editor, the user will be greeted with the `Save and exit/ Exit without saving/Close` options as usual. Rather, I modified the `Save Layout Modal` in the `graph/index.html` file which is not used for the time being. What I understood from the issue [#392](https://github.com/Murali-group/GraphSpace/issues/392) is that we wanted to reduce the no. of clicks required to save the layout that a user generates.
What was happening earlier:
`User generated the layout->click **Use Layout Editor** button->click **Exit Layout Editor** button->click **Save and Exit** button`
What happens now:
`User generated the layout->click **Save Layout** button->click **Save** button and its done.`

Reply:
Makes sense. 
Thanks for the clarification.